.github/workflows/sycl-precommit.yml

```diff
@@ -128,15 +128,40 @@ jobs:
       merge_ref: ''
 
       sycl_toolchain_artifact: sycl_linux_default
-      sycl_toolchain_archive: ${{ needs.build.outputs.artifact_archive_name }}
-      sycl_toolchain_decompress_command: ${{ needs.build.outputs.artifact_decompress_command }}
+      sycl_toolchain_archive: ${{ needs.build_linux.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.build_linux.outputs.artifact_decompress_command }}
+
+  build_win:
```

Comment:
Why are you removing the split between lin/win tasks? I would have expected you to link the commit that introduced the split in the PR's description together with your arguments on why you think that was wrong.

Also, you're not removing `sycl-windows-precommit.yml`, why?

Reply:
Also, even if we decide to keep that change, it should be done in a separate PR and not here.

Reply:
> Why are you removing the split between lin/win tasks? I would have expected you to link the commit that introduced the split in the PR's description together with your arguments on why you think that was wrong.

So people can see all precommit tests together. It never occurred to me that they might have been together at one point and then separated out, so I didn't consider something was done "wrong", thus no need to focus on

> Also, you're not removing sycl-windows-precommit.yml, why?

That's a mistake. Will fix. 

> Also, even if we decide to keep that change, it should be done in a separate PR and not here.

Which change? Why does it need a separate merge request?

Reply:
> Which change?

Merging win/lin into a single YML, obviously...

> Why does it need a separate merge request?

Because that would be an atomic change and per good software development practices PRs have to represent atomic changes, not a bunch of different fixes at once.

> It never occurred to me that they might have been together at one point and then separated out

They were. If you decide to merge them back, please make it clear in your new atomic PR why original reasoning doesn't take place anymore.

Reply:
> obviously

Not obviously. Don't patronise me. Your description was unclear and I'm not a mind reader

Reply:
My first comment in this thread was attributed to a *single* change:

> Why are you removing the split between lin/win tasks?

The second (and the only other comment at that time) in this thread was

> if we decide to keep that change...

What else could it possibly refer to, in your opinion?